homeassistant/components/intesishome/climate.py

```diff
@@ -0,0 +1,405 @@
+"""Support for IntesisHome and airconwithme Smart AC Controllers."""
+import logging
+from random import randrange
+
+from pyintesishome import AuthenticationError, ConnectionError, IntesisHome
```

Comment:
We should import `ConnectionError` as a different name to not overwrite the builtin exception.